library/core/src/ops/deref.rs

```diff
@@ -365,16 +365,18 @@ unsafe impl<T: ?Sized> DerefPure for &mut T {}
 /// }
 /// ```
 #[lang = "receiver"]
-#[unstable(feature = "arbitrary_self_types", issue = "44874")]
+#[cfg(not(bootstrap))]
```

Comment:
What is the point of totally cfg'ing out the receiver trait here when `bootstrap` is enabled?

Reply:
I _think_ this was necessary in the past because the bootstrap compiler didn't understand the `lang_item`. I'm assuming that may not be the case any longer. I'll try removing it. Maybe it'll help with the CI problem.